docs/guide/hello.md

```diff
@@ -14,7 +14,7 @@ In the previous chapter you've learned how to install [Ignite CLI](https://githu
 This series of tutorials is based on a specific version of Ignite CLI, so be sure to install the correct version. For example, to install Ignite CLI v0.20.0 use the following command:
 
 ```bash
-curl https://get.ignite.com/cli@v0.20.0! | bash
+curl https://get.ignite.com/cli! | bash
```

Comment:
```suggestion
curl https://get.ignite.com/cli@v0.21.0! | bash

```

Reply:
if we mention installing a specific version, let's give a specific version, otherwise the latest non-released development version is installed